geoalchemy2/_functions_helpers.py

```diff
@@ -4,16 +4,22 @@
 from typing import Union
 
 
+def _wrap_docstring(docstring: str) -> str:
```

Comment:
this handles wrapping plus the explicit newlines in the docstrings

Reply:
nice! :star_struck: